### PR TITLE
[Chef]Turn off the Basic Information Reachable attribute 

### PR DESCRIPTION
**Pull request by erwinpan1**
Following #26331, Turn off the Basic Information Reachable attribute in Chef devices

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -79,6 +79,22 @@ jobs:
                       #       https://github.com/project-chip/connectedhomeip/issues/19169
                       #       https://github.com/project-chip/connectedhomeip/issues/22640
                       if [ "$idl_file" = './examples/all-clusters-app/all-clusters-common/all-clusters-app.matter' ]; then continue; fi
+                      if [ "$idl_file" = './examples/chef/devices/rootnode_contactsensor_lFAGG1bfRO.matter' ]; then continue; fi
```
Comment (andy31415) [inline .github/workflows/lint.yml:82]: why? These are conformance lint tests that should generally be enabled.

**Review by andy31415 — CHANGES_REQUESTED** [review]
This PR adds exceptions to the linter. This seems incorrect and should be explained.

If there are bugs, ideally we should fix the linter and not allow potentially invalid matter files.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (erwinpan1) [reply .github/workflows/lint.yml]: Sorry. I misunderstood this. I've rolled this back.

**Review by andy31415 — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]
Thank you!

**Pull request merged**